### PR TITLE
update SSL config for NGINX 

### PR DESCRIPTION
**Pull request by ddragosd**
The configuration for NGINX has changed recently, or more precisely how the certs are generated via `genssl.sh` and this PR updates the configuration to take the latest changes into account.

### CODE REVIEW

**Comment by ddragosd** [comment]
Referencing upstream changes : https://github.com/apache/incubator-openwhisk/pull/2427

**Comment by ddragosd** [comment]
@jeremiaswerner since you've merged the apache/incubator-openwhisk#2427 in the upstream can you review this as well ?

File: docker-compose/Makefile [hunk 1]
```diff
@@ -64,7 +64,7 @@ check-required-ports:
 .PHONY: setup
 setup:
 	mkdir -p ~/tmp/openwhisk/apigateway/ssl
-	cd $(PROJECT_HOME)/ansible/roles/nginx/files/ && ./genssl.sh $(DOCKER_HOST_IP)
+	cd $(PROJECT_HOME)/ansible/roles/nginx/files/ && ./genssl.sh $(DOCKER_HOST_IP) server
```
Comment (ddragosd) [inline docker-compose/Makefile]: @jeremiaswerner ^^^

File: docker-compose/apigateway/conf/whisk-docker-compose.conf [hunk 2]
```diff
@@ -12,8 +12,8 @@ server {
 
   ssl_session_cache    shared:SSL:1m;
   ssl_session_timeout  10m;
-  ssl_certificate      /etc/ssl/openwhisk-cert.pem;
-  ssl_certificate_key  /etc/ssl/openwhisk-key.pem;
+  ssl_certificate      /etc/ssl/openwhisk-server-cert.pem;
+  ssl_certificate_key  /etc/ssl/openwhisk-server-key.pem;
```
Comment (ddragosd) [inline docker-compose/apigateway/conf/whisk-docker-compose.conf]: @jeremiaswerner ^^^

**Comment by rabbah** [comment]
he's out/vacation - generally LGTM.

**Comment by ddragosd** [comment]
thanks @rabbah . I've added you as reviewer and approve if you're ok with these changes.

**Comment by ddragosd** [comment]
🎉 build passing now. Thanks @rabbah for taking the time to review this so quickly.

**Pull request merged**